src/DynamoDbQueryBuilder.php

```diff
@@ -41,6 +41,30 @@ class DynamoDbQueryBuilder
     protected $client;
 
     /**
+     * The methods that should be returned from query builder.
+     *
+     * @var array
+     */
+    protected $passthru = [
+        'insert', 'insertGetId', 'getBindings', 'toSql',
+        'exists', 'count', 'min', 'max', 'avg', 'sum', 'getConnection',
+    ];
```

Comment:
Actually, I don't think this is necessary ... the passthru is designed to pass calls down to the `Illuminate\Database\Query\Builder` for things such as aggregates.  This project does not have another query builder to pass calls down too ... so I'm removing the property and it's usage from the class.